src/parser.pegjs

```diff
@@ -26,16 +26,20 @@ comparison
 	= ( "=" alpha* / "!" ) "="
 
 argument
-	= "(" head:value tail:( "," value )* ")"  {return [head].concat((tail || []).map(item => item[1]));}
+	= "(" head:value tail:("," value )* ")"  {return [head].concat((tail || []).map(item => item[1]));}
 	/ value
 
 value
 	= c:( unreserved / pct_encoded )+  {return c.join('');}
 
+space
+ = " "
+
 unreserved
 	= alpha
 	/ digit
 	/ [-._~:]
+	/ string
```

Comment:
I think it's better not to add `string` here at the `unreserved` rule but above at the `value` rule instead. I've the impression that the `string` is at another level than the other characters here. And having `string` at the `unreserved` rule enables `string` in `selector` but it seems that the `string` rule isn't resolved to the content of the quotes but to the entire expression including the quotes then. E.g. a query `'"a selector"==VALUE'` creates an actual selector `'"a selector"'` instead of `'a selector'` as I would expect.